src/components/atoms/Input.svelte

```diff
@@ -4,11 +4,13 @@
   export let name;
```

Comment:
Why this is not initialized and the other ones do?

Reply:
Actually that component was not supposed to be on this PR. But I found needs to apply some changes to it for the profile page to use it, that's why it appears here. And I was trying to keep it as it was the most I could.